plugins/TodayAtMun/today.py

```diff
@@ -0,0 +1,80 @@
+from datetime import datetime, timedelta
+from json import load
+from pathlib import Path
+from typing import Dict
```

Comment:
wanna see if you can use type definitions for more places than just here? not a requirement in any way, but a tidy thing to do :)

Reply:
Sure, typing is new to me and was an after thought in the pl. Will take a look

Reply:
Increased this slightly particularly in the __init__.py. I looked over my other classes and I didn't see any opportunities. Of course if you have any suggestions that'd be great.